documentation/assertions/Promise/to-be-rejected.md

```diff
@@ -26,15 +26,13 @@ expected Promise (fulfilled) to be rejected
 You can assert the promise is rejected with a specific reason (error) by
 passing a second parameter:
 
-```javascript
+```javascript#async:true
 var promiseThatWillBeRejectedWithAReason = expect.promise(function (resolve, reject) {
     setTimeout(function () {
         reject(new Error('Oh dear'));
-    }, 1);
+    }, 10);
 });
```

Comment:
If you don't want to duplicate this code, you could also just create a function that will create the promise. I'm okay with both approaches.